src/components/utilities.js

```diff
@@ -27,7 +27,7 @@ export function ResultWrapper(props) {
     })
   }
   if (field.field === "id") {
-    const link_target = window.config.thingpage + "/" + parse(text)
+    const link_target = window.config.thingpage + "/" + value
```

Comment:
`text` at this point is already wrapped in a `<span>` highlighting it yellow, so that's the wrong thing to use in the link.  Use the original value.